### ISSUE TITLE
Cannot get WK 13 to publish on gitbook

### ISSUE THREAD

**Issue by drewdinger** [bigdatahci2015/forum#69]
Hey all,
I am trying to publish my group project stuff that is located in week 13 locally. When I do the command "git add build" nothing happens and it says everything is up to date, even though I have made changes. Has anyone else ran into this problem, or know of a fix? Thanks.

**Comment by drewdinger** [comment]
It says that I have untracked files??? I'm not sure what that means or how i can fix that. Any help would be appreciated.

**Comment by ZachLamb** [comment]
It means that you haven't added or committed those files. Git should tell you what files weren't added. Do a "git add path_to_file" , then a git commit ,and followed by your git push command. If that doesn't work, you could do a "git add --all" which will add EVERYTHING to your git repo ,but it will ensure everything is on there. Not the best approach ,but you're limited by time, this is your best option.

**Comment by drewdinger** [comment]
Hey thanks for the repy Zach. I am seeing what you are saying, but when I got to my fork of book 3 it has all of my files there uploaded to github, even the untracked ones that it was complaining about. Yet on my gitbook page it is still not showing the changes.

**Comment by drewdinger** [comment]
but now when I do a git commit it says I am on the branch master, that my branch is up to date with origin/master, and there is nothing to commit.

**Comment by ZachLamb** [comment]
Have you tried this: git subtree push --prefix build origin gh-pages ?

**Comment by drewdinger** [comment]
yeah, it counts all the way to n 37 and says that everything is up-to-date

**Comment by ZachLamb** [comment]
If nothing is showing up on your github.io/book3 ,then open up the Chrome Inspector tool in it ,and it might show you an error.

**Comment by drewdinger** [comment]
Yeah, thanks for trying. No luck, the inspector doesn't show any errors.

**Comment by drewdinger** [comment]
Do you know of a way I can just start over? Get rid of the changes I made in week 13, and just re add the files I edited. Is that an option? Sorry, I am not very good with Git.

**Comment by drewdinger** [comment]
I got it. Somehow there was a difference between my build folder and my src folder for week 13. I changed the folder in the build subdirectory and everything published smoothly. Thank you Zach for your help, I greatly appreciate it.
